platformio/package/manifest/schema.py

```diff
@@ -15,13 +15,14 @@
 import requests
 import semantic_version
 from marshmallow import Schema, ValidationError, fields, validate, validates
+from marshmallow import __version_info__ as marshmallow_version
 
 from platformio.package.exception import ManifestValidationError
 from platformio.util import memoized
 
 
 class StrictSchema(Schema):
-    def handle_error(self, error, data):
+    def handle_error(self, error, data, *, _many, **_kwargs):
```

Comment:
Python 2.7 doesn't have keyword only arguments 

https://www.python.org/dev/peps/pep-3102/
```suggestion
    def handle_error(self, error, data, _many, **_kwargs):
```